Library/Models/Template.cs

```diff
@@ -0,0 +1,44 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Nfield.Models
+{
+    /// <summary>
+    /// Holds all properties of a template
+    /// </summary>
+    public class Template
+    {
+
+        /// <summary>
+        /// Unique ID of the template
+        /// </summary>
+        [JsonProperty]
```

Comment:
why this attribute? Same for `Theme.Id`

Reply:
I see, but for which of [these cases](https://www.newtonsoft.com/json/help/html/serializationattributes.htm) do you need this attribute here?

```
- By default, the JSON property will have the same name as the .NET property. This attribute allows the name to be customized.
- JsonPropertyAttribute indicates that a property should be serialized when member serialization is set to opt-in.
- It includes non-public properties in serialization and deserialization.
- It can be used to customize type name, reference, null, and default value handling for the property value.
- It can be used to customize the NamingStrategy of the serialized property name.
- It can be used to customize the property's collection items JsonConverter, type name handling, and reference handling.
```